packages/react/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@novu/react",
+  "version": "2.0.0-canary.0",
```

Comment:
We don't want 2.0.0 here, right?

Reply:
afaik, it gets overridden by our script during the release and we maintain a same release version throughout the monorepo.
@LetItRock can we have different release number here and will it increment independently of the monorepo version when we release builds?

Reply:
Lerna allows us to keep different versions independent. But all the packages in our monorepo as synced with the same version and I don't think that we want to break this consistency now.